src/main/java/com/tg5/domain/Event.java

```diff
@@ -15,14 +15,20 @@ public class Event implements Serializable {
     private static final long serialVersionUID = 1L;
 
     @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @GeneratedValue
```

Comment:
Let's just keep entities with IDENTITY strategy, prof seems to like it.

Reply:
For MySQL it does something different, but for postgres,  it seems is equivalent to auto. Do we have to add to all entities